class Error(Exception):
    pass

class DuplicateRecord(Exception):
    pass